.github/workflows/main.yml

```diff
@@ -55,10 +55,23 @@ jobs:
             libxkbcommon-dev libdrm-dev libgbm-dev libgl1-mesa-dev libgles2-mesa-dev \
             libegl1-mesa-dev libdbus-1-dev libibus-1.0-dev libudev-dev fcitx-libs-dev
 
+    - name: Setup extra Ubuntu 20.04 dependencies
+      if: matrix.platform.os == 'ubuntu-20.04'
+      run: |
+        sudo apt-get install pulseaudio
+        systemctl --user start pulseaudio.service
+        systemctl --user start pulseaudio.socket
+
     - name: Setup extra Ubuntu 22.04 dependencies
       if: matrix.platform.os == 'ubuntu-22.04'
       run: |
-        sudo apt-get install libpipewire-0.3-dev libdecor-0-dev
+        sudo apt-get install libpipewire-0.3-dev libdecor-0-dev pipewire-media-session- wireplumber \
+        pipewire-audio-client-libraries
+        systemctl --user --now enable wireplumber.service
+        systemctl --user --now enable pipewire.service
+        systemctl --user --now enable pipewire.socket
+        systemctl --user --now enable pipewire-pulse.service
+        systemctl --user --now enable pipewire-pulse.socket
```

Comment:
Are these needed?
CTest runs the tests with `SDL_AUDIODRIVER=dummy SDL_VIDEODRIVER=dummy`,

Reply:
Some of the audio tests (the ones which were failing) check all of the drivers, regardless of which one is selected. We either have to start the services they expect, or drop the tests.